ibis/expr/types/relations.py

```diff
@@ -864,6 +864,45 @@ def fillna(
                 )
         return ops.FillNa(self, replacements).to_expr()
 
+    def unpack(self, *columns: str) -> Table:
+        """Project the struct fields `columns` the table.
+
+        Existing fields are retained in the projection.
+
+        Returns
+        -------
+        Table
+            The child table with struct fields in `columns` projected in.
+
+        Examples
+        --------
+        >>> schema = dict(a="struct<b: float, c: string>", d="string")
+        >>> t = ibis.table(schema, name="t")
+        >>> t
+        UnboundTable: t
+          a struct<b: float64, c: string>
+          d string
+        >>> t.unpack("a")
```

Comment:
If given a nested struct, could this also be make to take `"a.b"` to unpack the fields found at `t.a.b` into `t`?

Reply:
Is this any less confusing than `t.a.b.unpack()`?

Reply:
Let's avoid adding the `.` for now and wait to see how this gets used.